src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -31,8 +31,8 @@ public PathResponse findPaths(long source, long target) {
         Station startStation = findStationById(source);
         Station endStation = findStationById(target);
         List<Line> lines = findLines();
-        PathFinder pathFinder = PathFinder.ofList(lines);
-        Path shortestPath = pathFinder.findShortestPath(startStation, endStation);
+        DijkstraPathFinder dijkstraPathFinder = DijkstraPathFinder.ofList(lines);
+        Path shortestPath = dijkstraPathFinder.findPath(startStation, endStation);
```

Comment:
이전 피드백 잊지 않고 반영 잘해주셨네요 👍
아래처럼 `구현체`에 의존하기 보다는 `인터페이스`에 의존하도록 하면 좋겠네요 :)

```suggestion
        PathFinder pathFinder = DijkstraPathFinder.ofList(lines);
        Path shortestPath = pathFinder.findPath(startStation, endStation);
```